ydb/core/tx/datashard/export_common.cpp

```diff
@@ -59,7 +59,12 @@ TMaybe<Ydb::Table::CreateTableRequest> GenYdbScheme(
         return Nothing();
     }
 
-    FillIndexDescription(scheme, tableDesc, mkqlKeyType);
+    try {
+        FillIndexDescription(scheme, tableDesc);
+    } catch (const yexception&) {
+        return Nothing();
+    }
```

Comment:
NIT: There is no need to separate try catch for FillTableBoundary and FillIndexDescription